test/_generate_expected_node.js

```diff
@@ -63,7 +63,7 @@ const fileNames = files.slice(start, end || files.length).map((file) => {
     }
     console.log('Begin writing...');
     images.filter(i => i).forEach(([fileName, image]) => {
-      const expectedPath = `${__dirname}/expected/${fileName}.png`;
+      const expectedPath = `${__dirname}/expected/${path.basename(fileName)}.png`;
```

Comment:
`fileName` is a fully absoluted path here so you need to basename it @brettz9 

before
```
Getting svg buffers...
Loading image buffers...
Trying canvas for /Users/glerne19/Projects/canvg/svgs/24.svg
Begin writing...
Writing: /Users/gabe/Projects/canvg/test/expected//Users/gabe/Projects/canvg/svgs/24.svg.png...
Finished processing!
```

after
```
Getting svg buffers...
Loading image buffers...
Trying canvas for /Users/gabe/Projects/canvg/svgs/24.svg
Begin writing...
Writing: /Users/gabe/Projects/canvg/test/expected/24.svg.png...
Finished processing!
```

Reply:
Ah, apologies...